markdown/chapters/univariate-analysis/main.rmd

```diff
@@ -0,0 +1,1672 @@
+  ylab("Value")+
+  xlab("Plot Type")
+```
+
+Now consider the response distribution for the logarithmically-scaled plot, after removing the two responses of "10^15" and "10^9", as well as the values at 1000.
+
+```{r, fig.cap = "Density plot showing distributions of responses regarding the logarithmic scaled plot, after removing values of greater or equal to 1000"}
+
+
+ggplot() +
+  geom_density(data = as.data.frame(logarithmic), aes(x=logarithmic, col = "Logarithmic"))+
+  labs(x="Response", y="Density")+
+  scale_colour_manual(name = " ", breaks = brks[3], values = vals[3])+
+  theme_classic()
+
+```
+We see that after removing the outlying values, taken as any value above 1000, the distribution is positively skewed and centres around 40 with a tail up to 150. The summary statistics for the outlier removed set shows a mean of 37.45, with a range of $[9, 150]$ and median of 35. The IQR shows that $50\%$ of these values lie in the range $[22.5, 40]$, meaning we appear to potentially have a greater underestimation than for the other two plots, also shown by the mean.
+
+Finally consider the three distributions together.
+
+```{r, fig.cap = "Density plot showing distributions of responses regarding all three plots, after removing values greater or equal to 1000 from the responses fore the logarithmic plot"}
+
+brks <- c("Control", "Truncated", "Logarithmic")
+vals <- c("#1c9e77", "#d95f02", "#7570b3")
+
+ggplot() +
+  geom_density(data = as.data.frame(control), aes(x=control, col = "Control"))+
+  geom_density(data = as.data.frame(truncated), aes(x=truncated, col = "Truncated"))+
+  geom_density(data = as.data.frame(logarithmic), aes(x=logarithmic, col = "Logarithmic"))+
+  labs(x="Response", y="Density")+
+  scale_colour_manual(name = " ", breaks = brks, values = vals)+
+  theme_classic()
+
+```
+
+The distribution looks remarkably different to the other two, with much less height and a wider spread. 
+
+Overall, it appears that the use of the truncated scale had little impact on judging the height of an individual bar as compared to a control with no scale alterations, and if anything marginally improved the respondents' ability to judge the height. It has also been observed that the scale by default ending at 40 while the bar height is at 42 leads to some underestimation in the height for both the control and truncated bar plots.
+
+Now considering the logarithmic-scale plot, The Python default of standard form notation appears to have confused certain respondents, who are perhaps not as used to seeing this notation, and there was a very large range in the responses along with one person not even entering a number, but rather stating that they "Don't know", and another stating they believed the value was "Next to none". The "Next to none" entry is very subjective, but could potentially be be assumed as a value close to 0, once again maybe as a result of standard form being less well known to this respondent.
```

Comment:
```suggestion
Now considering the logarithmic-scale plot, The Python default of standard form notation appears to have confused certain respondents, who are perhaps not as used to seeing this notation, and there was a large range in the responses along with one person not even entering a number, but rather stating that they "Don't know", and another stating they believed the value was "Next to none". The "Next to none" entry is  subjective, but could potentially be be assumed as a value close to 0, once again maybe as a result of standard form being less well known to this respondent.
```

Reply:
> close to 0, once again maybe as a result of standard form being less well known to this respondent.

It *could* be argued that `next to none` is actually an indication of a very good understanding of standard form.